src/utils.js

```diff
@@ -8,6 +8,16 @@ import Image from 'components/Image';
 import Anchor from 'components/Anchor';
 
 
+export const getFormattedDateString = (dateString) => {
+  return new Date(dateString).toLocaleDateString();
```

Comment:
Not sure if we want to change this to `return new Date(dateString).toLocaleDateString('nl-NL');` to force the `locale`.  As it is now the date for me was display as DD/MM/YYYY even when changing my browser settings to Dutch.  If we force the `locale` then the dates will be displayed as DD-MM-YYYY.

Reply:
that's not what I said in my review though, I said to use the react-intl package tooling. They have a `FormattedDate` component - which you can't use here because this is not a react component, but you should be able to use the underlying imperative API: https://formatjs.io/docs/react-intl/api#formatdate

Reply:
Needs to use the `react-intl` functions.